### PR TITLE
Lint on Travis.

### PR DESCRIPTION
**Pull request by Ms2ger**
This change is [Reviewable](https://reviewable.io/reviews/w3c/csswg-test/1120)

### CODE REVIEW

**Comment by gsnedders** [comment]
Do we want to, in some way, to ignore `dist` for example? Probably?

**Comment by gsnedders** [comment]
We probably also want the same extension whitelist. And we probably should optimise the case where the whitelist ignores `*` to not bother touching the file (given parsing it is just wasted time!).

**Comment by gsnedders** [comment]
Also what does this actually leave enabled? Or is it entirely useless at the moment?

**Comment by Ms2ger** [comment]
I was mostly looking for the relative-testharness-urls lint I added (though that hasn't landed here yet).

**Comment by gsnedders** [comment]
@Ms2ger r?

File: lint.whitelist [hunk 1]
```diff
@@ -0,0 +1,195 @@
+# File containing whiteslist for lint errors
```
Comment (Ms2ger) [inline lint.whitelist]: "whiteslist"

**Pull request merged**